assets/javascript/game.js

```diff
@@ -0,0 +1,126 @@
+// Global Variables section
+//___________________________________________________________________
+// Tech Company names
+var TechCompany = ["apple", "samsung", "microsoft", "alphabet", "intel", "ibm", "facebook","tencent", "oracle"];
+var nextWord = "";  //word being guessed
+var letters = [];     //letters in word
+var blanksNum = 0;    //calculate the number of blanks left based on number of letters left in word
+var currentGuess = [];// correct + blanks
+var wrongLetters = [];//incorrect guesses
+
+var guessesWrong = 0;
+var guessesLeft = 0;
+var numWins = 0;
+var numLosses = 0;
+
+
+//___________________________________________________________________
+// Functions section
+//___________________________________________________________________
+
```

Comment:
Generally comments are just for your own use so in your final commits you should remove them before you push it up.

Reply:
Also its the same thing with ``console.log()``. You should just use those for your own use while you're in development, but remove them at the end.

Reply:
OK